cluster-scope/overlays/moc/zero/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ kind: Kustomization
 resources:
```

Comment:
Missing the `metallb-system` namespace in this file

Reply:
@larsks I'm still missing the namespace imported in here from the base. Am I looking right? Sorry getting lost in the PR a bit :smile:  

Also the podsecuritypolicies are missing from here.

Reply:
@larsks ^ the podsecuritypolicies are missing as well

Reply:
Aaaand I've added those, also.